docs/ref/models/querysets.txt

```diff
@@ -119,6 +119,21 @@ described here.
     unpickle a queryset in a Django version that is different than the one in
     which it was pickled.
 
+Generated queries
```

Comment:
This might be better in `topics/db/queries.txt`. Originally I thought we would be documenting a queryset method, but it's actually `queryset.query.__str__()`. Any opinion?

We should also document that there's no way to get the SQL for terminal queryset methods (at least this restriction will apply to 1.11) and defer the second commit to a separate PR.